wagtail/admin/templates/wagtailadmin/shared/breadcrumb-next.html

```diff
@@ -7,51 +7,62 @@
-            aria-label="{% trans 'Toggle breadcrumbs' %}"
-            aria-expanded="false"
-        >
-            {% icon name="breadcrumb-expand" class_name="w-w-4 w-h-4" %}
-        </button>
-
+        {% if not is_expanded %}
+            <button
+                type="button"
+                data-toggle-breadcrumbs
+                class="w-flex w-items-center w-justify-center w-box-border w-ml-0 w-p-4 w-w-[50px] w-h-full w-bg-transparent w-text-grey-400 w-transition hover:w-scale-110 hover:w-text-primary w-outline-offset-inside"
+                aria-label="{% trans 'Toggle breadcrumbs' %}"
+                aria-expanded="false"
+            >
+                {% icon name="breadcrumb-expand" class_name="w-w-4 w-h-4" %}
+            </button>
+        {% endif %}
         <div class="w-relative w-h-[50px] w-mr-4 w-top-0 w-z-20 w-flex w-items-center w-flex-row w-flex-1 sm:w-flex-none w-transition w-duration-300">
             <nav class="w-flex w-items-center w-flex-row w-h-full"
                 aria-label="{% trans 'Breadcrumb' %}">
                 <ol class="w-flex w-flex-row w-justify-start w-items-center w-h-full w-pl-0 w-my-0 w-gap-2 sm:w-gap-0 sm:w-space-x-2">
                     {% for page in pages %}
                         {% if page.is_root %}
-                            <li class="{{ breadcrumb_item_classes }} w-max-w-0" data-breadcrumb-item hidden>
-                                <a class="{{ breadcrumb_link_classes }}" href="{% url 'wagtailadmin_explore_root' %}">
+                            <li
+                                class="{{ breadcrumb_item_classes }} w-max-w-0 {% if is_expanded %}w-max-w-4xl{% endif %}"
+                                data-breadcrumb-item
+                                {% if not is_expanded %}hidden{% endif %}
+                            >
+                                <a
+                                    class="{{ breadcrumb_link_classes }}"
+                                    href="{% if url_root_name %}{% url url_root_name %}{% else %}{% url url_name page.id %}{% endif %}{{ querystring_value }}"
+                                >
                                     {% trans "Root" %}
                                 </a>
                                 {% icon name="arrow-right" class_name=icon_classes %}
                             </li>
                         {% elif forloop.first %}
                             {# For limited-permission users whose breadcrumb starts further down from the root #}
-                            <li class="{{ breadcrumb_item_classes }} w-max-w-0" data-breadcrumb-item hidden>
-                                <a class="{{ breadcrumb_link_classes }}" href="{% url 'wagtailadmin_explore' page.id %}">
+                            <li class="{{ breadcrumb_item_classes }} w-max-w-0 {% if is_collapsible %}" data-breadcrumb-item hidden {% else %}w-max-w-4xl" data-breadcrumb-item{% endif %}>
```

Comment:
This one is still using the `is_collapsible`

Reply:
Great catch!